app/views/layouts/tylium/_sidebar.html.erb

```diff
@@ -8,7 +8,7 @@
         <i class="fa fa-tachometer sidebar-link-icon"></i> <span class="sidebar-link-label">Dashboard</span>
       <% end %>
     <% end %>
-    <%= content_tag :li, :class => controller_path == 'issues' || controller_path == 'issues/merge' ? 'active' : '' do %>
+    <%= content_tag :li, :class => controller_path.include?('issues') ? 'active' : '' do %>
```

Comment:
Why isn't the issues/merge path relevant anymore?

Reply:
It's not that it's no longer relevant. ` controller_path.include?('issues')` picks up `issues` in `issues/merge`. 

This change was done to get the correct nav-item to be active for all issues related views without adding a `... || controller_path == 'issues/evidence' || controller_path == 'issues/whatever_else_in_the_future'`